docusaurus/docs/reactnative/common-content/core-components/channel/props/max_file_size_to_upload_in_mb.mdx

```diff
@@ -0,0 +1,5 @@
+Limit for maximum size of the file/image that can be attached to a message while uploading. The upper limit and the default value is 100 MB.
+
+| Type   | Default |
+| ------ | ------- |
+| number | 100     |
```

Comment:
since 100mb is a hardcoded limit on the BE side.. I strongly suggest to make it simple to not have a prop for this and just hardcode 100mb inside our SDK also

cc @vanGalilea

Reply:
Added a documentation for the max limit being 100MB

Reply:
This was introduced in favour that the customer can provide lower limits. I don't see any other way of achieving lower limits other than a prop.

Reply:
Do we have a customer that asked for this feat? (setting a lower max upload size for files) @khushal87